src/js/lib/monster.ui.js

```diff
@@ -1280,7 +1280,35 @@ define(function(require) {
 		customValidationInitialized: false,
 
 		initCustomValidation: function() {
-			var localization = monster.apps.core.i18n.active().validation,
+			var validationI18n = monster.apps.core.i18n.active().validation,
+				defaultRulesI18n = _.get(validationI18n, 'defaultRules'),
+				getRuleI18n = _.partial(function(i18n, ruleId) {
+					return _
+						.chain([
+							'customRules',
+							'defaultRules'
+						])
+						.map(_.partial(_.ary(_.get, 2), i18n))
+						.find(_.flow(
+							_.partial(_.ary(_.get, 2), _, 'ruleId'),
+							_.negate(_.isUndefined)
+						))
+						.value();
+				}, validationI18n),
+				getRuleMessageForPlural = function(plural, ruleId) {
+					return _
+						.chain(ruleId)
+						.thru(getRuleI18n)
+						.get(plural)
+						.value();
+				},
+				getRuleMessageForOne = _.flow(
+					_.over([
+						_.partial(getRuleMessageForPlural, 'one'),
+						getRuleI18n
+					]),
+					_.partial(_.find, _.isString)
```

Comment:
In this command, the first parameter for `lodash#find` needs to be the array to search over:
```suggestion
					_.partial(_.find, _, _.isString)
```